tools/CustomMSBuild/Versioning.props

```diff
@@ -11,9 +11,9 @@
   <!-- For NuGet Package Dependencies -->
   <PropertyGroup>
     <ODataClientPackageDependency>[7.5.1.0, 8.0.0)</ODataClientPackageDependency>
-    <ExtensionsDependencyInjectionDependency>[2.1.0,4.0.0)</ExtensionsDependencyInjectionDependency>
-    <ExtensionsOptionsDependency>[2.1.0,4.0.0)</ExtensionsOptionsDependency>
-    <ExtensionsHttpDependency>[2.1.0,4.0.0)</ExtensionsHttpDependency>
+    <ExtensionsDependencyInjectionDependency>[2.1.0,5.0.0)</ExtensionsDependencyInjectionDependency>
+    <ExtensionsOptionsDependency>[2.1.0,5.0.0)</ExtensionsOptionsDependency>
```

Comment:
Change to 
2.1 <=  version < 5.0 doesn't work?

Reply:
I just saw your commit with [2.1.0,) on the dependency range.
Thank you.